_posts/2021-07-09-part 19.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+author: Anthony
+---
+# Anthony's Blog
+Anthony's Adventures Around ISD!
+---
+
+title: Day 19!
+date:  2021-07-09
+---
+Hello again!! Troubleshooting... do you know what that means? Solving issues so my blog doesn't break. Issues range from tiny issues to whole pages not working. It's intresting how many issues you can run into in just one day! I had issue today my blog posts not showing up. I was stuck on it for a while actually. It took two hours but I finally found the problem. I noticed the my links looked different. The name of the file was causing it not show up. I did a lot of testing and eventually found it.My last few minutes are coming up. Until next time.
+
+Here's a picture, can you spot the diffence?
```

Comment:
I don't see your what i've learned page. Is that supposed to be included in this PR?